Nightly jobs on master branch, in Travis CI, are failing, due to timeout(s)

The ```master```[0] build's jobs for ```nightly```[1] are failing, like so:

```
=================================== FAILURES ===================================
_______________________ test_tracking_protection_shield ________________________
foxpuppet = <foxpuppet.foxpuppet.FoxPuppet object at 0x7efdab43f250>
selenium = <selenium.webdriver.firefox.webdriver.WebDriver (session="576eca7a-c994-46bf-ba42-9e3171ff37c2")>
    @pytest.mark.firefox_preferences({'privacy.trackingprotection.enabled': True})
    def test_tracking_protection_shield(foxpuppet, selenium):
        """Tests if the tracking protection icon displays."""
        browser = foxpuppet.browser
        assert not browser.navbar.is_tracking_shield_displayed
        selenium.get('https://www.washingtonpost.com/')
        WebDriverWait(selenium, timeout=5).until(
>           lambda _: browser.navbar.is_tracking_shield_displayed)
tests/test_browser_model.py:55: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <selenium.webdriver.support.wait.WebDriverWait (session="576eca7a-c994-46bf-ba42-9e3171ff37c2")>
method = <function <lambda> at 0x7efdab740ed8>, message = ''
    def until(self, method, message=''):
        """Calls the method provided with the driver as an argument until the \
            return value is not False."""
        screen = None
        stacktrace = None
    
        end_time = time.time() + self._timeout
        while True:
            try:
                value = method(self._driver)
                if value:
                    return value
            except self._ignored_exceptions as exc:
                screen = getattr(exc, 'screen', None)
                stacktrace = getattr(exc, 'stacktrace', None)
            time.sleep(self._poll)
            if time.time() > end_time:
                break
>       raise TimeoutException(message, screen, stacktrace)
E       TimeoutException: Message:
.tox/py27/lib/python2.7/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException
```

[0] https://travis-ci.org/mozilla/FoxPuppet/builds/406067197
[1] https://travis-ci.org/mozilla/FoxPuppet/jobs/406067206

--- a/foxpuppet/windows/browser/navbar.py
+++ b/foxpuppet/windows/browser/navbar.py
@@ -20,7 +20,8 @@ class NavBar(Region):
 
     """
 
-    _tracking_protection_shield_locator = (By.ID, 'tracking-protection-icon')
+    _tracking_protection_shield_locator = (
+        By.ID, 'tracking-protection-icon-box')
 
     @property
     def is_tracking_shield_displayed(self):
@@ -31,6 +32,10 @@ class NavBar(Region):
 
         """
         with self.selenium.context(self.selenium.CONTEXT_CHROME):
-            el = self.selenium.find_element(
-                *self._tracking_protection_shield_locator)
+            if self.window.firefox_version >= 63:  # Bug 1471713
+                el = self.root.find_element(
+                    *self._tracking_protection_shield_locator)
+            else:
+                el = self.root.find_element(
+                    By.ID, 'tracking-protection-icon')
             return bool(el.get_attribute('state'))

